src/pykeen/contrib/lightning.py

```diff
@@ -157,6 +157,8 @@ def train_dataloader(self):
     def val_dataloader(self):
```

Comment:
can we get a type annotation on this function? or is it goofy from the base class?

Reply:
[d40438c](https://github.com/pykeen/pykeen/pull/1158/commits/d40438c917b675450bc0bd620bf9b436c2f21a07)